pkgs/applications/video/obs-studio/v4l2sink.nix

```diff
@@ -1,42 +1,49 @@
-{ stdenv, fetchFromGitHub
-, cmake, pkgconfig, wrapQtAppsHook
-, obs-studio }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, qt5
+, obs-studio
+}:
 
-stdenv.mkDerivation {
-  pname = "obs-v4l2sink-unstable";
-  version = "20181012";
+stdenv.mkDerivation rec {
+  pname = "obs-v4l2sink";
+  version = "0.1.0";
```

Comment:
This seems to have been a downgrade. For software that has been so long without a release, following `HEAD` might be preferable.

Reply:
The changes on master are not very important. There's a couple localization fixes and only one build-related change (CatxFish/obs-v4l2sink#1ec3c8ada0e1040d867ce567f177be55cd278378), but it doesn't seem to be critical.

Personally I would rather stay on the release rev.

Reply:
I reverted the revision change.

Reply:
Right, with this diff https://github.com/CatxFish/obs-v4l2sink/compare/0.1.0...master, I would prefer the tag as well. Though the downgrade should be mentioned in the commit message.

Reply:
OBS doesn't seem to recognize the plugin since I upgraded to 20.03 (even with the current derivation on master).

Reply:
Everything is working again. The downgrade is now documented in the commit message.